Menus/MainMenu/Scripts/MainMenuToggle.cs

```diff
@@ -0,0 +1,54 @@
+﻿#if UNITY_EDITOR
+using System;
+using UnityEditor.Experimental.EditorVR;
+using UnityEditor.Experimental.EditorVR.Modules;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+#if INCLUDE_TEXT_MESH_PRO
+using TMPro;
+#endif
+
+[assembly: OptionalDependency("TMPro.TextMeshProUGUI", "INCLUDE_TEXT_MESH_PRO")]
+
+namespace UnityEditor.Experimental.EditorVR.Menus
+{
+    sealed class MainMenuToggle : MainMenuSelectable, IRayEnterHandler, IRayExitHandler, IPointerClickHandler
+    {
+        [SerializeField]
+        Toggle m_Toggle;
+
+        public Toggle toggle { get { return m_Toggle; } }
+
+        public event Action<Transform> hovered;
+        public event Action<Transform> clicked;
+
+        new void Awake()
+        {
+            m_Selectable = m_Toggle;
+            m_OriginalColor = m_Toggle.targetGraphic.color;
+        }
+
+        public void OnRayEnter(RayEventData eventData)
+        {
+#if INCLUDE_TEXT_MESH_PRO
```

Comment:
Don't think this is #if block is needed?

Reply:
Yeah, there are no TMP references in this entire class. Looks like it's left over from before the refactor to MainMenuSelectable